### ISSUE TITLE
input required returning just  a sting

### ISSUE THREAD

**Issue by levansuper** [evantahler/actionhero#631]
hello

I have some complex system and need to return custom error object when the input is required and is not passed to the method. 

example of what i'm getting now :
{
    "error": "Error: token is a required parameter for this action",
    "serverInformation": { ...  },
    "requesterInformation": { ... }
}

example output that is needed: 
{
 "error":  { message : "token is a required parameter for this action",  code : 101} 
    "serverInformation": { ...  },
    "requesterInformation": { ... }
}

and i want to be able to make some changes to the error object before it's returned to the user

**Comment by evantahler** [comment]
This is a duplicate of https://github.com/evantahler/actionhero/issues/620
(which was solved by https://github.com/evantahler/actionhero/pull/624)

tldr; it's coming in version 11.0.0
